.github/workflows/android-release.yml

```diff
@@ -14,21 +14,27 @@ jobs:
       BUILDTYPE: Release
       IS_LOCAL_DEVELOPMENT: false
     steps:
-
       - uses: actions/checkout@v4
         with:
           submodules: recursive
           fetch-depth: 0
 
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v1.14
+        with:
+          cmake-version: "3.19.x"
```

Comment:
CMake 3.27.6 is included by default.

https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#tools

Reply:
This just unifies it with the CI. I can revert it from both.

Reply:
Ah! Yes please.

Reply:
Done.